mapsforge-map/src/main/java/org/mapsforge/map/datastore/MultiMapDataStore.java

```diff
@@ -152,15 +152,15 @@ public MapReadResult readLabels(Tile tile) {
                 }
                 return null;
             case RETURN_ALL:
-                return readLabels(tile, false);
+                return readLabelsPriv(tile);
             case DEDUPLICATE:
-                return readLabels(tile, true);
+                return readLabelsPriv(tile).deduplicate();
         }
         throw new IllegalStateException("Invalid data policy for multi map database");
 
     }
 
-    private MapReadResult readLabels(Tile tile, boolean deduplicate) {
+    private MapReadResult readLabelsPriv(Tile tile) {
```

Comment:
@Sublimis Let's use a more understandable name in these methods.

Either the whole word "private" or better a different name, like `processLabels`, `searchLabels`, etc.